src/valhalla_run_isochrone.cc

```diff
@@ -79,6 +83,12 @@ int main(int argc, char* argv[]) {
   // Process json request
   Api request;
   ParseApi(json_str, valhalla::Options::isochrone, request);
+
+#ifdef ENABLE_GDAL
+  if (request.options().format() == Options_Format_geotiff) {
+    GDALRegister_GTiff();
+  }
+#endif
```

Comment:
haha that's noble but honestly, these executables are hardly used by anyone (AFAIK). dave uses the matrix one now and then for perf testing I believe, but other than that they're not too useful IMO. more for future reference:)